lib/versions_repository.py

```diff
@@ -15,12 +15,12 @@ def setup_versions_repository(config):
     Clone and checkout the versions repository and halt execution if
     anything fails.
     """
-    path, dir_name = os.path.split(
+    path, _ = os.path.split(
```

Comment:
I prefer to not alter how parent directory path is obtained in this patch. This is an improvement which could be done in a separate patch.